app/views/layouts/application.html.slim

```diff
@@ -20,11 +20,16 @@ html.govuk-template.app-html-class lang="en"
     meta content=asset_pack_path("media/images/govuk-opengraph-image.png") property="og:image"
     = csrf_meta_tags
   body class=body_class data-vacancy_state=(@vacancy.present? ? @vacancy.state : "create")
+    - if show_test_variant(:"2021_02_cookie_consent_test", :modal) && cookie_preferences_show?
+      .modal-container
     = render partial: "shared/google_tag_manager_body" if cookies["consented-to-cookies"] == "yes"
     = javascript_tag(nonce: true) do
       | document.body.className = ((document.body.className) ? document.body.className + " js-enabled" : "js-enabled");
     = render "shared/skip_links"
-    = render "layouts/cookies_banner" unless controller_name == "cookies_preferences" || cookies_preference_set?
+    = render(AbTest::CookiesBannerComponent.new(create_path: create_cookies_preferences_path(cookies_consent: "yes", **utm_parameters), preferences_path: cookies_preferences_path(**utm_parameters)).with_variant(:bottom)) if show_test_variant(:"2021_02_cookie_consent_test", :bottom) && !cookie_preferences_hidden?
```

Comment:
Could we just do this in one go? e.g.:

```ruby
render(AbTest::CookiesBannerComponent.new(...).with_variant(ab_variant_for(:"2021_02_cookie_consent_test"))) if cookie_preferences_show?
```

Reply:
what i would have ideally like to do but the presence of another variant lower in the markup makes it tricky

Reply:
Gotcha. Maybe another alternative could be:

```ruby
- unless cookie_preferences_hidden? || current_variant?(:"2021_02_cookie_consent_test", :top)
  = render(AbTest::CookiesBannerComponent.new(...).with_variant(ab_variant_for(:"2021_02_cookie_consent_test")))
```